pages/events/index.vue

```diff
@@ -332,6 +334,14 @@ export default class extends QiskitPage {
     }
   }
 
+  &__empty-card-description {
```

Comment:
How is this change related to this fix?

Reply:
I took commits directly from the advocates branch. This issue was fixed in that branch as a side effect of fixing other issues.
That fixes changed the empty card too, and it was necessary to fix it. I do it this way to be faster and make easier the future merge with that branch.